src/main/java/duke/storage/Storage.java

```diff
@@ -0,0 +1,66 @@
+package duke.storage;
+
+import duke.commands.CommandHandler;
+import duke.exception.IllegalTaskCommandException;
+import duke.parser.Parser;
+import duke.task.TaskList;
+import duke.ui.Ui;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Scanner;
+
+public class Storage {
+    private static final String FILE_PATH = "Duke_Tasks.txt";
+
+    public static void saveTasks() throws IOException {
+        File file = new File(FILE_PATH);
+
+        FileWriter writer = new FileWriter(file);
+        writer.write(TaskList.convertToFileInput());
+        writer.close();
+    }
+
+    public static void loadTasks() throws IOException, IllegalTaskCommandException {
+        File file = new File(FILE_PATH);
+
+        if (file.exists()) {
+            Scanner scanner = new Scanner(file);
+
+            while (scanner.hasNext()) {
+                String[] fileInput = scanner.nextLine().split(",");
+                String loadedCommand = Parser.getLoadedCommand(fileInput);
+
+                switch (fileInput[0].strip()) {
+                case "T":
+                    CommandHandler.executeTodo(loadedCommand);

```

Comment:
Command execution here? (maybe I have misunderstood here, or it could be an issue with the method name)

Reply:
I have used the same method to parse both my file data and user input into the Task. Would it be better to split them up? Because I feel that it is logical to have both calling onto the same method itself